src/frontend/qt_sdl/main.cpp

```diff
@@ -99,6 +101,51 @@
 
 // TODO: uniform variable spelling
 
+const QStringList NdsRomMimeTypes { "application/x-nintendo-ds-rom" };
+const QStringList NdsRomExtensions { ".nds", ".srl", ".dsi", ".ids" };
+
+const QStringList GbaRomMimeTypes { "application/x-gba-rom" };
+const QStringList GbaRomExtensions { ".gba", ".agb" };
+
+const QStringList ArchiveMimeTypes {
+#ifdef ARCHIVE_SUPPORT_ENABLED
+    "application/zip",
+    "application/x-7z-compressed",
+    "application/vnd.rar", // *.rar
+    "application/x-tar",
+
+    "application/x-compressed-tar", // *.tar.gz
+    "application/x-xz-compressed-tar",
+    "application/x-bzip-compressed-tar",
+    "application/x-lz4-compressed-tar",
+    "application/x-zstd-compressed-tar",
+
+    "application/x-tarz", // *.tar.Z
+    "application/x-lzip-compressed-tar",
+    "application/x-lzma-compressed-tar",
+    "application/x-lrzip-compressed-tar",
+    "application/x-tzo", // *.tar.lzo
+#endif
+};
+const QStringList ArchiveExtensions {
```

Comment:
same here

Reply:
Fixed in 04061eda0b08c0f51428fb4dc93cc2f7e8f37686.